rktlet/runtime/helpers.go

```diff
@@ -387,6 +407,19 @@ func generateAppSandboxCommand(req *runtimeApi.RunPodSandboxRequest, uuidfile, s
 		stage1Name = val
 	}
 
+	// Get log directory from run pod sandbox request.
+	var logDirectory string
+	if req.Config.LogDirectory == "" {
+		logDirectory = path.Join("/var/log/pods", req.Config.Metadata.Uid)
+		glog.Warningf("log directory not specified in sandbox config, which shouldn't happen when running rktlet with a real kubernetes installation, setting it to %s", logDirectory)
+		os.MkdirAll(logDirectory, 0770)
```

Comment:
I don't like ignoring the error here. Can we either add `error` to this function or do this outside?